### PR TITLE
Add rule delete group or user

### PR DESCRIPTION
**Pull request by tuanhxh1**
Delete User or Group Using Userdel or Groupdel

### CODE REVIEW

File: rules/linux/process_creation/proc_creation_lnx_userdel.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+title: Delete User or Group Using Userdel or Groupdel
+id: 8a46f16c-8c4c-82d1-b121-0fdd3ba70a84
+status: experimental
+description: Detects usage of the "userdel" or "groupdel" binary to delete user or delete group
+references:
+    - https://cve.mitre.org/cgi-bin/cvename.cgi?name=2019-19882
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_userdel.yml:6]: Why this ref ?

Comment (tuanhxh1) [reply rules/linux/process_creation/proc_creation_lnx_userdel.yml:6]: I see this ref related to process groupdel and userdel

Comment (frack113) [reply rules/linux/process_creation/proc_creation_lnx_userdel.yml:6]: This CVE is for shadow 4.8 and misconfigured setuid.

I am the operator who receives this alert.
I read the references, how I make the link between this CVE and this basic detection on a process execution.

If it is a privileged escalation that we want to detect, there is some information missing in the detection.
This is more a "simple"  [T1531 Account Access Removal ](https://attack.mitre.org/techniques/T1531/)

File: rules/linux/process_creation/proc_creation_lnx_userdel.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+title: Delete User or Group Using Userdel or Groupdel
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_userdel.yml:1]:
```suggestion
title: Deleting an Account or a Group in CLI
```

File: rules/linux/process_creation/proc_creation_lnx_userdel.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+id: 8a46f16c-8c4c-82d1-b121-0fdd3ba70a84
+status: experimental
+description: Detects usage of the "userdel" or "groupdel" binary to delete user or delete group
+references:
+    - https://cve.mitre.org/cgi-bin/cvename.cgi?name=2019-19882
+    - https://www.cyberciti.biz/faq/linux-remove-user-command/
+author: Tuan Le (NCSGroup)
+date: 2022/12/26
+tags:
+    - attack.impact
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_userdel.yml:11]:
```suggestion
    - attack.impact
    - attack.t1531
```

File: rules/linux/process_creation/proc_creation_lnx_userdel.yml (hunk 2 above)
Comment (tuanhxh1) [reply rules/linux/process_creation/proc_creation_lnx_userdel.yml:1]: Look good, thanks

File: rules/linux/process_creation/proc_creation_lnx_userdel.yml (hunk 3 above)
Comment (tuanhxh1) [reply rules/linux/process_creation/proc_creation_lnx_userdel.yml:11]: Look good, thanks

File: rules/linux/process_creation/proc_creation_lnx_userdel.yml (hunk 1 above)
Comment (tuanhxh1) [reply rules/linux/process_creation/proc_creation_lnx_userdel.yml:6]: Ok, thank you

File: rules/linux/process_creation/proc_creation_lnx_userdel.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+title: Deleting an Account or a Group in CLI
+id: 8a46f16c-8c4c-82d1-b121-0fdd3ba70a84
+status: experimental
+description: Detects usage of the "userdel" or "groupdel" binary to delete user or delete group
+references:
+    - https://cve.mitre.org/cgi-bin/cvename.cgi?name=2019-19882
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_userdel.yml:6]:
```suggestion
```
Did not find a APT or malware report using them. This command are use to remove the trace of these ...

Comment (tuanhxh1) [reply rules/linux/process_creation/proc_creation_lnx_userdel.yml:6]: I deleted old ref and update new ref

**Review by frack113 — APPROVED** [review]

**Comment by nasbench** [comment]
Since the binaries are targeting 2 different things (groups/users) I think it's best to split them to make FP handling easier.

I did that and I went ahead and added some more metadata info and changed the title to be read as an alert instead of an action.

**Review by nasbench — APPROVED** [review]

**Review by frack113 — APPROVED** [review]

**Pull request merged**